### PR TITLE
[Nala]: Bump to latest

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Changes https://github.com/brave/leo/compare/eef21eefd16f8aa564d49abcb7f0af4d30cb84e2...96fbe5c38ebea8e5d8cacc30a94ea32e92639cd6
- [[Link]: lower specificity of link selectors](https://github.com/brave/leo/commit/e514d7a5380ce099624af23186fd8df4209bc5d1)
- [[Button]: add solid transparent border as default](https://github.com/brave/leo/commit/a15774413a8f8f7c9e8d0f7885222e95d47a7740)
- [[Meta]: removes console log in android icon generation](https://github.com/brave/leo/commit/bf713bd1ebbeba76945f1c50a3093db597dea4c4)
- [[Button]: fix button height consistency](https://github.com/brave/leo/commit/0cbe77a3e466ae47a2bb45aa671481abf101d924)
- [[Button]: ensure proper padding when contents is only a text node](https://github.com/brave/leo/commit/b517bc1313af3ee879279b375da9ca632944cdc2)
- [[Collapse]: nests selectors to avoid poluting global scope for tailwind](https://github.com/brave/leo/commit/8ace9061780dc91a452b667535ed9c22f0334186)
- [[Toggle]: Don't drag on right click](https://github.com/brave/leo/commit/b44fca8ce3eb18a1d51d6ef05bd07549af142017)
- [[Icons]: Update icons from Figma](https://github.com/brave/leo/commit/740f245db739d9a785355d3df55bd39172e48438)
- [[React]: Change default export name for easier importing](https://github.com/brave/leo/commit/97d066d551bf352624f56139792ed9bc3538061d)
- [[Button]: rename types to avoid svelte2tsx adding underscore](https://github.com/brave/leo/commit/32f4f11ab6af8973b4fc9e9eaef6241246cdb879)
- [[React]: Add slot support](https://github.com/brave/leo/commit/3a6f66609cd54a7fc97ff9b7f3376051b6bba17b)
- [[Icon]: Ensure consuming apps receive Icon's IconName type](https://github.com/brave/leo/commit/dac66a2c115b30cc58a08aa0a584ebcc9517c582)
- [[Button]: Make empty string truthy in HTML attributes](https://github.com/brave/leo/commit/b7e0d7dafbabb1ccaf8163868795dfe9c99d4eae)
- [[Input]: Placeholder color should follow text color](https://github.com/brave/leo/commit/a9a5a1069210fca04eebe57555225f425c932d5e)
- [[Toggle]: Shorten animation, remove shadow](https://github.com/brave/leo/commit/9a91cfffd991e169ab97c226dbbbbb89e3dff774)
- [[Tokens] Tailwind: fixes color references in shadows](https://github.com/brave/leo/commit/96fbe5c38ebea8e5d8cacc30a94ea32e92639cd6)

### CODE REVIEW

File: components/ai_chat/resources/page/components/page_context_toggle/index.tsx [hunk 1]
```diff
@@ -22,8 +22,7 @@ function PageContextToggle() {
     context.updateShouldSendPageContents(checked)
   }
 
-  const toggleTooltipVisibility = (e: any) => {
-    e.preventDefault()
```
Comment (fallaciousreasoning) [inline components/ai_chat/resources/page/components/page_context_toggle/index.tsx:26]: @nullhook could you confirm this won't break anything?

File: components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx [hunk 2]
```diff
@@ -14,8 +14,7 @@ import getPageHandlerInstance from '../../api/page_handler'
 export default function LongConversationInfo() {
   const context = React.useContext(DataContext)
 
-  const handleClearChat = (e: any) => {
-    e.preventDefault()
```
Comment (fallaciousreasoning) [inline components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx:18]: Can you confirm this won't break something @nullhook?

**Review by kdenhartog — APPROVED** [review]
No transitive dep updates. LGTM

File: components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx (hunk 2 above)
Comment (nullhook) [reply components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx:18]: just tested and it doesn't break

File: components/ai_chat/resources/page/components/page_context_toggle/index.tsx (hunk 1 above)
Comment (nullhook) [reply components/ai_chat/resources/page/components/page_context_toggle/index.tsx:26]: this works!

**Pull request merged**